erp-client/src/app/models/user.model.ts

```diff
@@ -2,5 +2,8 @@ export class User {
   id: number;
   email: string;
   username: string;
-  isAdmin: string;
+  password: string;
+  signature: string;
+  isAdmin: boolean;
```

Comment:
I noticed this was a string in dev/master. It's a boolean in the DB and server, so I figured it should be a boolean here too. But, I'm interested to check on this and see if it breaks anything. I imagine there will have to be a few tweaks yet to get this PR to play nicely with Tucker's auth work, so I'll make sure to check on this.